src/models/common/decorators/fieldConfig/classFieldConfig.decorator.ts

```diff
@@ -0,0 +1,51 @@
+import 'reflect-metadata';
+import { getFieldConfig, IPropFieldConfigInfo } from './fieldConfig.decorator';
+
+const fieldConfigMetadataKey = Symbol('fieldconfigclass');
+type KeyValueDict = Record<string, unknown>;
+const target = {};
+
+function getFieldConfigs(object: KeyValueDict): IPropFieldConfigInfo[] {
+  const ret = [];
+  for (const prop in object) {
+    const fieldConfigMap = getFieldConfig(object, prop);
+    if (fieldConfigMap) {
+      ret.push({
+        prop: prop,
+        ...fieldConfigMap,
+      });
+    }
+  }
+  return ret;
+}
+
+function getFieldConfigClassesInfo(): IFieldConfigClassInfo[] | undefined {
+  return Reflect.getMetadata(fieldConfigMetadataKey, target) as IFieldConfigClassInfo[];
+}
+
+export interface IFieldConfigClassInfo {
+  name: string;
+  fields: IPropFieldConfigInfo[];
+}
+
+export function fieldConfigClass(): ClassDecorator {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  return <TFunction extends Function>(classCtr: TFunction): TFunction => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
+    const classInstance = new (classCtr as any)();
```

Comment:
add the no unexpected any to the lint ignore comment

Reply:
Done, as well for graphQL class decorator